report-modules-plugins.py

```diff
@@ -569,7 +569,7 @@ def process_playbooks_path(playbooks_path, ctx):
         if os.path.isfile(itempath):
             process_ansible_file(itempath, ctx)
     # treat it like a role - look for vars/, tasks/, etc.
-    process_role(playbooks_path, False, ctx)
+    process_role(playbooks_path, True, ctx)
```

Comment:
That's it?  And this works?  If it doesn't work with big collections like community.general, that's ok, as long as it works with all roles in CI.  We can then worry about adding support for other collections later.